kubeflow/aws/OWNERS

```diff
@@ -1,3 +1,8 @@
 approvers:
   - jeffwan
-  - jlewi
\ No newline at end of file
+  - jlewi
```

Comment:
No-need to add these folks; You can actually go ahead and remove me as well. OWNERS are automatically inherited (unless you explicitly add the option to disable it). 
That ensures all the OWNERs at the lower levels can act as a backstop to approve PRs.

The primary purpose of adding finer grained OWNERs files is to better assign reviews to people who have the most context.

All of the folks other than you are Googlers with little understanding of AWS :) So adding them as approvers doesn't really help because all we can do is pretty much rubber stamp these PRs since we don't have much understanding of AWS.

So a better thing to do would be to try to grow the set of approvers and reviewers by getting more folks in the community using AWS involved. Then you can review and approve each other's PRs.

@mameshini and the folks have been working a lot with Kubeflow on AWS. Maybe you could add them as approvers or reviewres?

Reply:
Sounds great! Thanks for the details. I will clean it up and add @mameshini as approver.

Reply:
Clean it up and hope to attract more folks.